buildSrc/src/main/kotlin/com/datadog/gradle/plugin/benchmark/ReviewBenchmarkResultsTask.kt

```diff
@@ -16,23 +19,28 @@ open class ReviewBenchmarkResultsTask : DefaultTask() {
     internal lateinit var extension: ReviewBenchmarkExtension
     internal lateinit var buildDir: File
 
-    private val visitor = BenchmarkJsonFileVisitor()
+    val visitor = BenchmarkJsonFileVisitor()
 
     init {
         group = "datadog"
-        description = "Review the benchmark results and ensure they are below the specified threshold"
+        description =
+            "Review the benchmark results and ensure they are below the specified threshold"
     }
 
     // region Task
 
     @TaskAction
     fun applyTask() {
         val targetDir = getInputDir()
+        val benchmarksResults = mutableMapOf<String, Long>()
         if (targetDir.exists() && targetDir.canRead()) {
             targetDir.listFiles()?.forEach {
-                processDeviceBenchmarks(it)
+                processDeviceBenchmarks(it, benchmarksResults)
```

Comment:
Doing this won't work if you have multiple device attached and running.
It'll first check all the standard threshold, and the last device result will overwrite previous results in the  `benchmarksResults` map, leading untrustable results